providers/js/fido-u2f-admin.js

```diff
@@ -1,32 +1,48 @@
 /* global u2f, u2fL10n */
 ( function( $ ) {
 	var $button = $( '#register_security_key' );
+	var $status_notice = $( '#security-keys-section .security-key-status' )
+	var u2f_supported = false;
+
+	$status_notice.text( u2fL10n.text.u2f_not_supported );
+
+	u2f.getApiVersion( function() {
+		u2f_supported = true;
```

Comment:
Both Firefox and Safari simply fail to respond with anything. We're assuming that any response is a good response.

Secondly, we do support all U2F operations even if this check fails.